src/test/groovy/info/solidsoft/gradle/pitest/PitestTaskConfigurationSpec.groovy

```diff
@@ -254,8 +258,8 @@ class PitestTaskConfigurationSpec extends BasicProjectBuilderSpec implements Wit
     }
 
     private Set<String> assembleSourceSetsClasspathByNameAsStringSet(String sourceSetName) {
-        return [new File(project.buildDir, "classes//java//${sourceSetName}"),
-                new File(project.buildDir, "resources//${sourceSetName}")
+        return [new File(project.buildDir, "classes/java/${sourceSetName}"),
+                new File(project.buildDir, "resources/${sourceSetName}")
```

Comment:
I believe that I had some problems (somewhere) with just a single `/` on Windows and `//` was fine on both *Unix and Windows. However, the tests passed on AppVeyor, so it doesn't seem to be a problem here...